### PR TITLE
feat: add main page layout, add bg for header and footer, temporar commented route buttons

### PR DESCRIPTION
**Pull request by turn2river**
w8ing for feedback. ty in advance

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Link } from "react-router-dom";
+
+
+interface ButtonProps {
+  buttonText: string;
+  link: string;
+}
+
+export const Button: React.FC<ButtonProps> = ({ buttonText, link }) => {
```
Comment (PavelTitov16) [inline src/components/Button.tsx]: I suggest to rename component. Button is a general name. in this case for example maybe call like NavButton

File: src/App.tsx [hunk 2]
```diff
@@ -31,6 +31,6 @@ export const App = () => {
       </Router>
       <Footer />
     </div>
-    
+
```
Comment (PavelTitov16) [inline src/App.tsx]: Why we need this empty line?

File: src/App.tsx [hunk 3]
```diff
@@ -10,10 +10,10 @@ import { Footer } from './components/Footer';
 
 export const App = () => {
   return (
-    <div className="app flex flex-col justify-between h-screen">
+    <div className="app flex flex-col justify-between h-screen bg-pageBg">
       <Header />
       <Router>
-        <nav>
+        {/* <nav>
```
Comment (PavelTitov16) [inline src/App.tsx]: Better to remove comments

File: src/components/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  return (
+    <>
+      <Link to={link} className='rounded-full bg-primary px-3.5 py-3.5 text-sm font-semibold text-white hover:bg-white hover:text-primary hover:outline hover:outline-primary focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600 duration-300'>{buttonText}</Link>
```
Comment (PavelTitov16) [inline src/components/Button.tsx]: Please check the nesting of the text inside the component to make it more readable. I'm not sure maybe in IDE it's fine - can be bug in GitHub

File: src/pages/HomePage.tsx [hunk 5]
```diff
@@ -1,32 +1,25 @@
-import { useSelector, useDispatch } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 
-import { RootState, increment, decrement } from '../store';
+import { Button } from '../components/Button.tsx'
 
-import reactLogo from '../assets/react.svg';
-import viteLogo from '/vite.svg';
+import pic from '../assets/homepage_main-pic.png'
```
Comment (PavelTitov16) [inline src/pages/HomePage.tsx]: Add semicolon in the end of the line please

File: src/pages/HomePage.tsx [hunk 6]
```diff
@@ -1,32 +1,25 @@
...
+import logo from '../assets/LOGO_big.svg'
```
Comment (PavelTitov16) [inline src/pages/HomePage.tsx]: Add semicolon in the end of the line too please

File: src/pages/HomePage.tsx [hunk 7]
```diff
@@ -1,32 +1,25 @@
...
 
 export const HomePage = () => {
-  const count = useSelector((state: RootState) => state.counter.value);
-  const dispatch = useDispatch();
-  const { t } = useTranslation();
+  const { t } = useTranslation()
```
Comment (PavelTitov16) [inline src/pages/HomePage.tsx]: Same request for semicolon

File: src/pages/HomePage.tsx [hunk 8]
```diff
@@ -1,32 +1,25 @@
-import { useSelector, useDispatch } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 
-import { RootState, increment, decrement } from '../store';
+import { Button } from '../components/Button.tsx'
```
Comment (PavelTitov16) [inline src/pages/HomePage.tsx]: Add semicolon in the end of the line too please

File: src/pages/HomePage.tsx [hunk 9]
```diff
@@ -1,32 +1,25 @@
...
-        <h2>{t('welcome')}</h2>
-      </div>
+      <main className='mx-auto rounded-[16px] bg-white flex justify-between overflow-hidden drop-shadow-xl '>
+        <div className='m-auto h-[305px] w-[336px] py-10 px-5 flex flex-col justify-between items-center'>
+          <img className='mx-auto' src={logo} alt="logo" />
+          <p>Lorem ipsum dolor sit amet pretium consectetur adipiscing elit. Lorem consectetur adipiscing elit. </p>
+          <Button buttonText={t('welcome')} link='/auth' />
+        </div>
+        <div className='m-auto min-h-[305px] w-[456px]'>
+          <img src={pic} alt="main picture" />
```
Comment (PavelTitov16) [inline src/pages/HomePage.tsx]: Why we paste an image into components folder?

File: src/pages/HomePage.tsx [hunk 10]
```diff
@@ -1,32 +1,25 @@
...
-        <button onClick={() => dispatch(decrement())}>-</button>
-        <span>{count}</span>
-        <button onClick={() => dispatch(increment())}>+</button>
-      </div>
-
-      <div>
-        <h2>{t('welcome')}</h2>
-      </div>
+      <main className='mx-auto rounded-[16px] bg-white flex justify-between overflow-hidden drop-shadow-xl '>
+        <div className='m-auto h-[305px] w-[336px] py-10 px-5 flex flex-col justify-between items-center'>
```
Comment (PavelTitov16) [inline src/pages/HomePage.tsx]: I suggest to store custom css rules into talwind config variables for their reusability

File: src/pages/HomePage.tsx [hunk 11]
```diff
@@ -1,32 +1,25 @@
...
+          <img className='mx-auto' src={logo} alt="logo" />
+          <p>Lorem ipsum dolor sit amet pretium consectetur adipiscing elit. Lorem consectetur adipiscing elit. </p>
+          <Button buttonText={t('welcome')} link='/auth' />
+        </div>
+        <div className='m-auto min-h-[305px] w-[456px]'>
```
Comment (PavelTitov16) [inline src/pages/HomePage.tsx]: like and example for placing height into talwind variables

**Review by PavelTitov16 — CHANGES_REQUESTED** [review]

File: src/pages/HomePage.tsx (hunk 9 above)
Comment (turn2river) [reply src/pages/HomePage.tsx:20]: decline, picture is in the assets folder

**Review by PavelTitov16 — APPROVED** [review]

**Review by KristiBo — APPROVED** [review]

**Pull request merged**